libcxxabi/test/native/x86_64-unknown-linux/lpstart-zero.pass.sh.s

```diff
@@ -0,0 +1,107 @@
+# RUN: %{cxx} %{flags} %s %{link_flags} -no-pie -o %t.exe
```

Comment:
We have test/native/AArch64, a directory which only encodes the arch name.
We can have test/native/x86_64 and use `// REQUIRES: target={{x86_64-.+-linux}}`

(We have `test/native/arm-linux-eabi/` with the full triple just because arm eabi is very different)